src/mpid/common/genq/mpidu_genq_common.h

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (C) by Argonne National Laboratory
+ *     See COPYRIGHT in top-level directory
+ */
+
+#ifndef MPIDU_GENQ_COMMON_H_INCLUDED
+#define MPIDU_GENQ_COMMON_H_INCLUDED
+
+#include <stdint.h>
+
+typedef void *(*MPIDU_genq_malloc_fn) (uintptr_t);
+typedef void (*MPIDU_genq_free_fn) (void *);
```

Comment:
Do we need these for this PR?

Reply:
No, it is only used in private pool. I will remove it.